.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+
+# Description
+<!--- Describe your changes to provide context for reviewrs -->
+
+# Changes
```

Comment:
cc @Ivansete-status @LNSD 

I decided to follow the structure @LNSD suggested, but keep the test and issue sections as comments - to suggest the PR submitter that it might be a good idea to provide these, but to keep the default PR template general since many PRs do not have issue to close and there are no manual tests required.